templates/includes/sidebar.html

```diff
@@ -1,4 +1,10 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li><a href="{% url 'taxi:driver-detail' pk=user.id %}">User: {{ user.username }}</a></li>
+    <li><a href="{% url "logout" %}">Logout</a></li>
+  {% else %}
+    <li><a href="{% url "login" %}">Login</a></li>
```

Comment:
In Django templates, it's recommended to use single quotes for URL tags to maintain consistency. Change `{% url "login" %}` to `{% url 'login' %}`.